stories/button.js

```diff
@@ -1,3 +1,4 @@
+// @flow
```

Comment:
I enabled it on stories in order to test flow as users uses stories as a guide ;)

Reply:
@MonPote we don't use flow (yet) on stories because storybook itself doesn't have flow bindings yet, so a lot of what's defined is just untyped - and it's less of a problem because stories are not production code.